.github/workflows/cross-compiles.yml

```diff
@@ -0,0 +1,33 @@
+name: Cross Compile
+
+on: [pull_request, push]
+
+jobs:
+  cross-compilation:
+    strategy:
+      fail-fast: false
+      matrix:
+        # The platform matrix strings are two fields separated with a colon.
+        # The first field is the package to install using apt.
+        # The second field is passed to Configure as command line arguments.
+        platform: [
+          'gcc-aarch64-linux-gnu:-cross-compile-prefix=aarch64-linux-gnu- linux-aarch64',
```

Comment:
A dash is missing in `-cross-compilation`

Reply:
Hmmmm.... how did this pass?

Reply:
It wasn't buliding that's why